ui/app/styles/components/empty-state-component.scss

```diff
@@ -53,9 +53,6 @@
   a,
   .link,
   a:not(.button):not(.file-delete-button):not(.tag) {
-    color: $blue;
```

Comment:
These overrid HDS styling for HDS elements yielded to empty state. Removing so styles align with HDS expectations: 
<img width="1112" alt="Screenshot 2023-12-05 at 1 15 57 PM" src="https://github.com/hashicorp/vault/assets/68122737/4a21b19e-079c-48e6-b8f4-791508146aed">